app/controllers/social_previews_controller.rb

```diff
@@ -61,7 +61,7 @@ def set_respond(template = nil)
       format.png do
         html = render_to_string(template, formats: :html, layout: false)
         redirect_to HtmlCssToImage.fetch_url(html: html, css: PNG_CSS,
-                                             google_fonts: "Roboto|Roboto+Condensed"), status: :found
+                                             google_fonts: I18n.t("social_previews_controller.fonts")), status: :found
```

Comment:
We don't need to translate the font names.

Reply:
I'm not very sure where the piece of code is working now, but does it generate image that contains text? If so, different languages might need different fonts because "Roboto" apparently does not cover worldwide.

Reply:
Ugh, yes, you're probably right on this one.